### PR TITLE
feat: #98 create AllUsersPage

### PR DESCRIPTION
**Pull request by Arterixs**
1. Add users page

### CODE REVIEW

File: mw-webapp/src/pages/usersPage/UsersPage.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Users page will contain all user tables and the filter/search panel
```
Comment (tritonJS826) [inline mw-webapp/src/pages/usersPage/UsersPage.tsx:2]: You violate encapsulation here and also brings
Duplication

File: mw-webapp/src/pages/usersPage/UsersPage.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+ *@public
```
Comment (tritonJS826) [inline mw-webapp/src/pages/usersPage/UsersPage.tsx:3]: Looks like we can use public almost everywhere, let's skip it

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/pages/usersPage/UsersPage.tsx (hunk 1 above)
Comment (Arterixs) [reply mw-webapp/src/pages/usersPage/UsersPage.tsx]: Done

File: mw-webapp/src/component/table/Table.tsx [hunk 3]
```diff
@@ -29,7 +28,7 @@ export const Table = () => {
     columns,
     getCoreRowModel: getCoreRowModel(),
   });
-
+  let keyCounter = 0;
```
Comment (tritonJS826) [inline mw-webapp/src/component/table/Table.tsx:31]: Hahaha) comon) https://react.dev/reference/react/useId

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/dataAccessLogic/getDayReports.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
Comment (tritonJS826) [inline mw-webapp/src/dataAccessLogic/getDayReports.ts:1]: !

File: mw-webapp/src/pages/usersPage/UsersPage.tsx [hunk 5]
```diff
@@ -0,0 +1,6 @@
+/**
+ * Page all users
```
Comment (tritonJS826) [inline mw-webapp/src/pages/usersPage/UsersPage.tsx:2]: if page named UserPage, why in description you have PageAllUsers? What the difference? It's not a literature project :) try to avoid synonims

File: mw-webapp/src/component/table/Table.tsx [hunk 6]
```diff
@@ -29,7 +28,7 @@ export const Table = () => {
...
   return (
     <div className={styles.container}>
       <table className={styles.table}>
```
Comment (tritonJS826) [inline mw-webapp/src/component/table/Table.tsx:34]: why did you change table if you task does not related to it and you did not use tale any where?

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/table/Table.tsx (hunk 3 above)
Comment (Arterixs) [reply mw-webapp/src/component/table/Table.tsx:31]: omg, this shouldn't have been here, fail)
image

File: mw-webapp/src/pages/usersPage/UsersPage.tsx (hunk 5 above)
Comment (Arterixs) [reply mw-webapp/src/pages/usersPage/UsersPage.tsx:2]: done)

File: mw-webapp/src/component/table/Table.tsx (hunk 6 above)
Comment (Arterixs) [reply mw-webapp/src/component/table/Table.tsx]: image

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/dataAccessLogic/getDayReports.ts [hunk 7]
```diff
@@ -27,6 +26,7 @@ export const getDayReports = async (): Promise<DayReport[]> => {
     return jobDone;
   });
 
+
   const plansForNextPeriod = firstReport.plansForNextPeriod
```
Comment (Ekaterina1994) [inline mw-webapp/src/dataAccessLogic/getDayReports.ts:30]: why 2 empty line?

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/dataAccessLogic/getDayReports.ts (hunk 7 above)
Comment (Arterixs) [reply mw-webapp/src/dataAccessLogic/getDayReports.ts:30]: The absence a prettier who would rule such things for me, fix)

**Review by Ekaterina1994 — APPROVED** [review]

**Review by SergioAJS — APPROVED** [review]
